packages/graphql-types-generator/test/__snapshots__/jsonOutput.ts.snap

```diff
@@ -49,7 +49,8 @@ exports[`JSON output should generate JSON output for a mutation with an enum and
 							\\"responseName\\": \\"__typename\\",
 							\\"fieldName\\": \\"__typename\\",
 							\\"type\\": \\"String!\\",
-							\\"isConditional\\": false
+							\\"isConditional\\": false,
+							\\"isDeprecated\\": false
```

Comment:
why is this extra field being added in the tests?

Reply:
`isDepreacted` became required on `GraphQLField` in version 15. So it now shows up on all fields including `__typename`. https://github.com/graphql/graphql-js/pull/2469

I don't see this causing any issue though.